### PR TITLE
Explain concurrency limitation

### PR DESCRIPTION
**Pull request by mnm678**
**Fixes issue #**: #49 

**Description of the changes being introduced by the pull request**: Adds a note to the readme explaining that the reference implementation is not thread safe.

### CODE REVIEW

**Review by lukpueh — APPROVED** [review]
LGTM.

Although this (rather technical) piece of information looks a bit lost in this (rather organizational) document. But I don't know a better place. So I'm fine with leaving it here for the time being.

File: README.md [hunk 1]
```diff
@@ -79,6 +79,12 @@ The issue tracker is intended for bug reports and to make feature requests.
 Major feature requests, such as design changes to the specification, should
 be proposed via a [TUF Augmentation Proposal](docs/TAP.rst) (TAP).
 
+Limitations
+-----------
+
+The reference implementation may behave unexpectedly when concurrently
+downloading from the same location to the same location.
```
Comment (trishankatdatadog) [inline README.md]: Sorry, could you clarify what you mean by "location" here? Thanks!

**Comment by mnm678** [comment]
Thanks for the reviews!

@trishankatdatadog I updated this per your comment

**Review by trishankatdatadog — APPROVED** [review]
💯

**Pull request merged**